embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/properties/SessionPropertiesServiceImpl.kt

```diff
@@ -0,0 +1,42 @@
+package io.embrace.android.embracesdk.session.properties
+
+import io.embrace.android.embracesdk.logging.InternalEmbraceLogger
+import io.embrace.android.embracesdk.logging.InternalStaticEmbraceLogger
+import io.embrace.android.embracesdk.ndk.NdkService
+
+internal class SessionPropertiesServiceImpl(
```

Comment:
If we want to be consistent with other services this should be called `EmbraceSessionProperties`.  Do we want to start naming them with Impl?

Reply:
Good point - I've renamed it for consistency.